### ISSUE TITLE
Spectrum all devices freeze

### ISSUE THREAD

**Issue by feckdej** [1kc/razer-macos#266]
Steps: 
- select "Spectrum all devices"
- all devices get synced with Spectrum 
- change a single device (example, a keyboard) to "Reactive"
- the keyboard sets to "Reactive"

Expected result:
The keyboard sets to "Reactive" while the rest of the devices stay on "Spectrum"

Actual result:
The keyboards set to "Reactive" but the rest of the devices freeze on the color they were cycling while in "Spectrum"

**Comment by dhobi** [comment]
The reason for this is that the "Spectrum all devices" is currently done as an animation and animations are stopped whenever another menu item is clicked.

This should be fixable by changing the call from "manual animation" to individual spectrum calls for each device at the same time.

**Comment by feckdej** [comment]
> This should be fixable by changing the call from "manual animation" to individual spectrum calls for each device at the same time.

Do you mean like selection Spectrum for each individual device? If that's the case, the problem with this is that the devices won't be color synched.

**Comment by dhobi** [comment]
Yes, but the code would have to do this as we (humans) are just not fast enough 😄

**Comment by dhobi** [comment]
This should be possible now as the spectrum is done with the decide specific spectrum command.
